apps/nextjs-website/src/components/atoms/CookieBannerScript/CookieBannerScript.tsx

```diff
@@ -0,0 +1,34 @@
+'use client';
+import { isProduction } from '@/config';
+
+function makeCookieScript(dataDomainScript?: string) {
+  return `
+    <script
+      src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js"
+      type="text/javascript"
+      charset="UTF-8"
+      data-domain-script="${dataDomainScript}"></script>
+    <script type="text/javascript"> function OptanonWrapper() { } </script>
+  `;
+}
+
+type CookieBannerScriptProps = {
+  cookieDomainScript?: string;
+};
+
+const CookieBannerScript = ({
+  cookieDomainScript,
+}: CookieBannerScriptProps) => {
+  const COOKIE_SCRIPT = makeCookieScript(cookieDomainScript);
+
+  return (
+    isProduction && (
```

Comment:
Use a ad-hoc feature flag to enable a given future instead a generic `isProduction` ([DEV-1207](https://pagopa.atlassian.net/browse/DEV-1207))